website/docs/migration/migrating-from-mongodb.md

```diff
@@ -15,13 +15,13 @@ In that case, you can migrate your data using MongoDB native tools such as `mong
 
 Before you go forward with the migration, you need to have the following:
 
-- MongoDB connection URI
+- Existing connection URI
 - FerretDB connection URI
 - MongoDB native tools
 
 ## Backup your MongoDB data
```

Comment:
Let's use "export" and "import" for headers, because "backup" is something you restore to the original place; it is not about migration